R/plot_cbass.R

```diff
@@ -985,15 +978,33 @@ cbass_path_plotly <- function(x,
       }
 
       path_static %>%
-        add_text(data = plot_frame_init,
-                 x = ~V1,
-                 y = ~V2,
-                 text = ~ObsLabel,
+        add_segments(data = plot_frame_init_dist[plot_frame_init_dist$dist>1,],
```

Comment:
Playing around with this, I don't see the "pointer segments" appearing. Do you have an example that shows them? I'm wondering if this `1` threshold should/could be configurable.

Reply:
> Playing around with this, I don't see the "pointer segments" appearing. Do you have an example that shows them? I'm wondering if this `1` threshold should/could be configurable.

I added pointers only for labels that are far away from points. Thus the plot won't be messy.

Reply:
I'm just trying to see an example of it myself - do you have one I can look at?

Reply:
Nevermind - I just needed a larger data set. Looks good.